mscore/CMakeLists.txt

```diff
@@ -715,7 +715,12 @@ else (MINGW)
            if (APPLE)
                set(PORTMIDI_LIB portmidi)
            else (APPLE)
-               set(PORTMIDI_LIB -lportmidi -lporttime) # Remove -lporttime on RPM-based systems where PortTime is part of PortMidi.
+               find_library(PORTTIME_LIB porttime)
+               if (PORTTIME_LIB)
+                   set(PORTMIDI_LIB portmidi porttime)
```

Comment:
Did you forget the `-l` in front of the libraries here?

Reply:
Most of the target_link_libraries() calls in this file doesn't use `-l`, it's more consistent this way (both are correct). It doesn't really matter, but the `if(APPLE)` branch doesn't use `-l` either.

Reply:
OK.